types/vosk/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "target": "es2016",
+        "lib": ["es6"],
+        "module": "commonjs",
+        "types": [],
+        "noEmit": true,
+        "esModuleInterop": true,
```

Comment:
Please avoid using "esModuleInterop` in DT packages: https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports